config/schema/analyze.schema.yml

```diff
@@ -0,0 +1,8 @@
+# Schema for the configuration files of the Analyze module.
+analyze.settings:
+  type: config_object
+  label: 'Analyze settings'
+  mapping:
+    status:
+      type: array
```

Comment:
I don't think this is a valid type. Think you want either `sequence` or `mapping` depending on if it's a list or dict.

Reply:
I've changed to mapping, but we can't go any deeper than that as the keys within "status" depend on which entities the site using the module has.